server/routes/upgradeMember.ts

```diff
@@ -28,7 +33,7 @@ const upgradeMember = async (models, req: UserRequest, res: Response, next: Next
   let member = await models.Role.findOne({
     where: {
       ...chainOrCommObj,
-      address_id: memberAddress.id,
+      address_id: memberAddress.address,
```

Comment:
Why is this changed?

Reply:
I'm not quite sure why that change is included. Sorry about this

Reply:
reverted in latest commit

Reply:
This still looks off to me, it looks like the address_id at the top was reverted but not this one?